javascript-data-model/order-history.js

```diff
@@ -0,0 +1,62 @@
+var orderHistory = [
+  {
+    order_number: '114-3941689-8772232',
+    order_placed: 'August 4, 2020',
+    ship_to: 'JS Masher',
+    order_details: {
```

Comment:
This property should store an array, not an object. In that array, you should have an object for each item in the order with all the data for that one item.